_scripts/evaluate_tools.py

```diff
@@ -26,11 +26,36 @@
         #                 " state code 04 and county code 001 ",
         # })
         #
-        model.invoke({
+        #model.invoke({
+        #   "chat_history": [],
+        #    "question": "Find me the SP document for corn, in Washington"
+        #})
+
+        output = model.invoke({
+            "chat_history": [],
+            "question": "Find me the SP document for peanut, Monroe, Missouri in 2024"
+        })
+        print(output)
+
+        output = model.invoke({
             "chat_history": [],
-            "question": "What is the percentage of policies indemnified for Washakie county in Wyoming "
-                        "for sugar beets under the APH program"
+            "question": "Find me the SP document for almond, san joaquim, california in 2024"
         })
+        print(output)
+
+        # todo overwrite the other context if no available documents?
```

Comment:
I see quite a few commented out lines of code... Do we want to keep them there to manually test? Or should we delete commented code?

Reply:
yes this is still manually tested. I will add more automatic testing this week